main.sql

```diff
@@ -190,12 +198,9 @@ CREATE TABLE mag.PaperAuthorAffiliations(
     OriginalAffiliation TEXT
   );
 
--- The input files may required some modifications 
-\! sed -e 's/\\/\\\\/g' < PaperAuthorAffiliations.txt > PaperAuthorAffiliations_.txt
```

Comment:
Redshift already escapes backslashes this way, leaving this line in doubles them all.